src/Uno.Extensions.Navigation.UI/Navigators/TypeExtensions.cs

```diff
@@ -6,12 +6,11 @@ public static class TypeExtensions
 {
 	public static ConstructorInfo? GetNavigationConstructor(this Type type, INavigator navigator, IServiceProvider services, out object[] constructorArguments)
 	{
-		var ctrs = type.GetConstructors();
-		foreach (var ctr in ctrs)
+		var ctr = type.GetConstructors().FirstOrDefault();
```

Comment:
Should we be able to configure which ctor to use?

Reply:
I can't see a strong use case for supporting multiple constructors for view model creation. In the case where different data can be passed into the same view model type, you could have multiple optional parameters on the constructor